e2e/tests/functional/plugins/lad/lad.e2e.spec.js

```diff
@@ -0,0 +1,135 @@
+            type: 'LAD Table',
+            name: "Test LAD Table"
+        });
+        await page.locator('[title="Edit"]').click();
+
+        await page.locator('.c-tree__item__view-control.c-disclosure-triangle').click();
+        await page.dragAndDrop('text=Test Sine Wave Generator', '.c-lad-table-wrapper');
+
+        await page.locator('button[title="Save"]').click();
+        await page.locator('text=Save and Finish Editing').click();
+
+        let getTelemValuePromise = new Promise(resolve => page.exposeFunction('getTelemValue', resolve));
+
+        await subscribeToTelemetry(page, sineWaveGeneratorIdentifier);
+
+        const subscribeTelemValue = await getTelemValuePromise;
+        const ladTableValuePromise = await page.waitForSelector(`text="${subscribeTelemValue}"`);
+        const ladTableValue = await ladTableValuePromise.textContent();
+
+        expect(ladTableValue).toBe(subscribeTelemValue);
+    });
+    test('telemetry value exactly matches latest telemetry value received in fixed time', async ({ page, openmctConfig }) => {
+        await page.goto('./', { waitUntil: 'networkidle' });
+        const { myItemsFolderName } = openmctConfig;
+
+        await page.locator('button:has-text("Fixed Timespan")').click();
+        await page.locator('[data-testid="conductor-modeOption-realtime"]').click();
+
+        await createDomainObjectWithDefaults(page, {
+            type: 'Sine Wave Generator',
+            name: "Test Sine Wave Generator"
+        });
+        const pageURL = page.url();
+        const sineWaveGeneratorIdentifier = pageURL.split('/').pop().split('?')[0];
+
+        await page.locator(`text=${myItemsFolderName}`).first().click();
+
+        await createDomainObjectWithDefaults(page, {
+            type: 'LAD Table',
+            name: "Test LAD Table"
+        });
+        await page.locator('[title="Edit"]').click();
+
+        await page.locator('.c-tree__item__view-control.c-disclosure-triangle').click();
+        await page.dragAndDrop('text=Test Sine Wave Generator', '.c-lad-table-wrapper');
+
+        await page.locator('button[title="Save"]').click();
+        await page.locator('text=Save and Finish Editing').click();
+
+        let getTelemValuePromise = new Promise(resolve => page.exposeFunction('getTelemValue', resolve));
+
+        await subscribeToTelemetry(page, sineWaveGeneratorIdentifier);
+        await changeToFixedTime(page);
+
+        const subscribeTelemValue = await getTelemValuePromise;
+        const ladTableValuePromise = await page.waitForSelector(`text="${subscribeTelemValue}"`);
+        const ladTableValue = await ladTableValuePromise.textContent();
+
+        expect(ladTableValue).toBe(subscribeTelemValue);
+    });
+});
+
+/**
+ * Util for subscribing to a telemetry object by object identifier
+ * @param {import('@playwright/test').Page} page
+ * @param {string} objectIdentifier identifier for object
+ */
+async function subscribeToTelemetry(page, objectIdentifier) {
```

Comment:
Should we generalize and extract this into `appActions` ?

Reply:
@unlikelyzero?

Reply:
i think so. The utility is clear and this is really clean. The only thing I would say is that we should update the JSDoc as much as possible

Reply:
After syncing with Khalid, I think it makes sense to keep in here and come up with another home for functions which tap into raw openmct functionality

OR

Move into `pluginFixtures.js`

Reply:
My vote is for the former for now as we are still working out the proper patterns for the fixtures.